QaMarsCompetition/QaMarsCompetition/PageObjects/EditSkill.cs

```diff
@@ -10,6 +10,7 @@ namespace QaMarsCompetition.PageObjects
 {
     public class EditSkill : CommonDriver
     {
+        public bool skilledited = false;
         //public IWebElement editshareskillbutton => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[1]/div/div[2]/a"));
```

Comment:
try to avoid commented out code if you dont need them or put a comment explaining why it is there

Reply:
sure aman. I made the changes.